### ISSUE TITLE
CT: migration errors for `*DipoleCheckbox.labelNode.visibleProperty`

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/molecule-polarity#166]
Migration errors in CT:

> molecule-polarity : migration : 1.2->main
http://128.138.93.172/continuous-testing/ct-snapshots/1693315654969/phet-io-wrappers/migration/?sim=molecule-polarity&locales=*&phetioDebug=true&phetioWrapperDebug=true&fuzz&oldVersion=1.2&migrationRate=10000&silent&wrapperContinuousTest=%7B%22test%22%3A%5B%22molecule-polarity%22%2C%22migration%22%2C%221.2-%3Emain%22%5D%2C%22snapshotName%22%3A%22snapshot-1693315654969%22%2C%22timestamp%22%3A1693315903290%7D
Assertion failed: These phetioIDs are not found in the new API. Either they were renamed incorrectly or were not migrated at all:
moleculePolarity.twoAtomsScreen.view.controlPanel.viewControls.bondDipoleCheckbox.labelNode.visibleProperty
moleculePolarity.threeAtomsScreen.view.controlPanel.viewControls.bondDipolesCheckbox.labelNode.visibleProperty
moleculePolarity.threeAtomsScreen.view.controlPanel.viewControls.molecularDipoleCheckbox.labelNode.visibleProperty

**Comment by pixelzoom** [comment]
First... I don't understand why the test is labeled as "migration : 1.2->main".    The most recent published version is 1.3, main is on 1.4, and migration-rules.ts only contains rules for 1.3.   Is there a bug here?

Second... These errors are occuring because @zepumph commented out migration rules in molecule-polarity. Pleae tell me if you're going to do something like this in one of the sims that's I'm responsible for.  I needlessly spent 30 minutes on this. 

```ts
   // These elements are no longer instrumented.
    // TODO: COMMENTED OUT ONLY FOR TESTING https://github.com/phetsims/phet-io/issues/1944, BRING THESE BACK PLEASE
    // new TandemFragmentDelete( 'moleculePolarity.twoAtomsScreen.view.controlPanel.viewControls.bondDipoleCheckbox.labelNode' ),
    // new TandemFragmentDelete( 'moleculePolarity.threeAtomsScreen.view.controlPanel.viewControls.bondDipolesCheckbox.labelNode' ),
    // new TandemFragmentDelete( 'moleculePolarity.threeAtomsScreen.view.controlPanel.viewControls.molecularDipoleCheckbox.labelNode' )
```

**Comment by pixelzoom** [comment]
> First... I don't understand why the test is labeled as "migration : 1.2->main".

In https://github.com/phetsims/phet-io/issues/1944#issuecomment-1697697736, @zepumph pointed to this in listContinuousTests.js:

```ts
// PhET-iO migration testing
[
  [ 'molecule-polarity', '1.2' ],
  [ 'graphing-quadratics', '1.2' ],
  [ 'natural-selection', '1
```

In Slack#dev-public, @zepumph said:
> When you migrate from 1.2 -> master, you go through all the possible versions, so it is actually still testing 1.3.
